ops/helper-scripts/az-func-deploy.sh

```diff
@@ -116,3 +105,14 @@ if [[ -n ${identities} ]]; then
 
     done
 fi
+
+# configure Application Settings
+if [[ -n ${app_settings} ]]; then
+    echo "Set Application Settings for ${app_name}"
+    for item in ${app_settings}; do
+        az functionapp config appsettings set -g $app_rg -n $app_name --settings "${item}" --query "[?name=='${item%=*}'].name | [0]" --output tsv
+    done
+fi
+
+# Gives some time for prior management operation to complete before going forward
+sleep 45s
```

Comment:
I don't think this will function quite as expected since each operation updating the function app config will cause a restart and this is outside of the for loop.

Reply:
I'm hoping that container will restart once for these set of changes. 

Looking through documentation and a search, I couldn't find any definitive answer on how Azure will behave. I think the best way to test this is to run this thru the pipeline.

Reply:
Add jira task CAMS-144 to address the looping. There should be a way to pass in a list of settings and have it execute once. We just need to adjust the output and make sure no secrets is logged